packages/codemod/src/upgrade-legacy.ts

```diff
@@ -317,8 +327,8 @@ const upgradeLegacy = async () => {
           })
           fs.writeFileSync(path.resolve(appDir, filename), program.toSource())
         } catch (e) {
-          log.error(`Error updating imports in the ${file}`)
-          throw new Error(e)
+          log.error(`Error updating imports in the ${filename}`)
+          throw new Error(e as string)
```

Comment:
Can you elaborate on why `as string` is needed here?

Reply:
Because otherwise it's unknown. For example you can check line 1340